cli/hzn.go

```diff
@@ -232,7 +234,7 @@ Environment Variables:
 
 	devServiceCmd := devCmd.Command("service", msgPrinter.Sprintf("For working with a service project."))
 	devServiceLogCmd := devServiceCmd.Command("log", msgPrinter.Sprintf("Show the container/system logs for a service."))
-	devServiceLogCmdServiceName := devServiceLogCmd.Flag("service", msgPrinter.Sprintf("(DEPRECATED) This flag is deprecated and is replaced by -c.")).Short('s').String()
```

Comment:
Still need to keep this line so that the old user will know what happened to this flag.  Since `devServiceLogCmdServiceName` is used next line, you can just do `devServiceLogCmd.Flag("service", ...)` without assigning it to any variable.

Reply:
addressed in most recent changes